frontend/src/app/general/breadcrumbs/breadcrumbs.component.html

```diff
@@ -5,6 +5,8 @@
 
 <ul>
```

Comment:
On the `<ul>` element add `class="break-all"` to also break words (try it out in the mobile view with project slug `asdfsdafsafat ft srafdasgasdfaq3fdasfsafasfasfasd`. It should wrap everywhere and not only at the spaces.